snap/snapcraft.yaml

```diff
@@ -72,350 +72,7 @@ parts:
     plugin: dump
     source: snap/local
 
-  musl:
-    source: https://musl.libc.org/releases/musl-1.2.3.tar.gz
-    source-type: tar
-    plugin: autotools
-    build-packages:
-      - gcc
-      - automake
-      - wget
-    override-build: |
-      set -ex
-
-      CUSTOM_CFLAGS=""
-      MACHINE_TYPE=$(uname -m)
-      if [ "${MACHINE_TYPE}" = "ppc64le" ]; then
-        MACHINE_TYPE="powerpc64le"
-        CUSTOM_CFLAGS="-mlong-double-64"
-      fi
-
-      ./configure CFLAGS="${CUSTOM_CFLAGS}"
-      make install
-
-      ln -s /usr/include/${MACHINE_TYPE}-linux-gnu/asm /usr/local/musl/include/asm
-      ln -s /usr/include/asm-generic /usr/local/musl/include/asm-generic
-      ln -s /usr/include/linux /usr/local/musl/include/linux
```

Comment:
So long thanks for the :fish: 

That was a solid couple of weeks figuring out how to get this to play nicely. Glad to see it go though!

Reply:
Yeah, it was great work, but glad we can see it go.